src/client/game.cpp

```diff
@@ -4148,7 +4148,7 @@ void Game::updateShadows()
 	float timeoftheday = getWickedTimeOfDay(in_timeofday);
 	bool is_day = timeoftheday > 0.25 && timeoftheday < 0.75;
 	bool is_shadow_visible = is_day ? sky->getSunVisible() : sky->getMoonVisible();
-	shadow->setShadowIntensity(is_shadow_visible ? client->getEnv().getLocalPlayer()->getLighting().shadow_intensity : 0.0f);
```

Comment:
The old way is intended. There will be more attributes to lighting in the future in addition to the shadow intensity.

Reply:
ok, let's revert this commit for now, and focus on the StaticObject + Mapblock refactor.
You can review again, that should fix vs build too (as it was stucked on this part), and merge as soon as you are okay. I'm continuing the refacto after